### ISSUE TITLE
Exception in class DTInterceptor if OPBEANS_SERVICES property is not set

### ISSUE THREAD

**Issue by artemnikitin** [elastic/opbeans-java#16]
If OPBEANS_SERVICES property is not set, then line):
```
hostList = env.getProperty("OPBEANS_SERVICES", "").split(","); 
```
(DTInterceptor:38) will return an array with 1 empty string. 
On line `if (hostList.length == 0)`(DTInterceptor:46) it will pass the check and will crash later with an error like:
```
2018-10-20 15:52:00.477 ERROR 91857 --- [nio-8080-exec-5] c.e.a.opbeans.controllers.DTInterceptor  : Returned unknown error 500 

org.springframework.web.client.ResourceAccessException: I/O error on GET request for "http://:3000/api/stats": Connection refused (Connection refused); nested exception is java.net.ConnectException: Connection refused (Connection refused)
```
Is it done intentional or this is a bug? If it's a bug, then I can submit a fix...

**Comment by felixbarny** [comment]
That sounds like a bug.

**Comment by artemnikitin** [comment]
Ok, when I will send a PR soon. I thought that it might be done intentionally because the app is for APM demo and you wanted to show error handling on APM side.

**Comment by felixbarny** [comment]
closed by #17
